Passive_Yaw_with_N_Waves/PassiveYawON/wecSimInputFile.m

```diff
@@ -0,0 +1,80 @@
+%% Simulation Data
+simu = simulationClass();               % Initialize Simulation Class
+simu.simMechanicsFile = 'OSWEC.slx';    % Specify Simulink Model File
+simu.mode = 'normal';                   % Specify Simulation Mode ('normal','accelerator','rapid-accelerator')
+simu.explorer='on';                    % Turn SimMechanics Explorer (on/off)
+simu.startTime = 0;                     % Simulation Start Time [s]
+simu.rampTime = .1;                    % Wave Ramp Time [s]
+simu.endTime=100;                       % Simulation End Time [s]        
+simu.solver = 'ode4';                   % simu.solver = 'ode4' for fixed step & simu.solver = 'ode45' for variable step 
+simu.dt = 0.1;                         % Simulation Time-Step [s]
+simu.cicEndTime = 40;                   % Specify CI Time [s]
+
+%% Wave Information
+
+waves(1) = waveClass('irregular'); 
+waves(1).height = 2;                     % Wave Height [m]
+waves(1).period = 3;                       % Wave Period [s]
+waves(1).spectrumType = 'PM';
+waves(1).direction = 0;
+waves(1).spread = 1;
```

Comment:
@salhus suggest removing line 20 and line 38 as the default spread option is already 1 so this is not necessary.

Reply:
- done, in the latest push